cranelift/object/src/backend.rs

```diff
@@ -342,22 +342,18 @@ impl Module for ObjectModule {
         }
         *defined = true;
 
+        let align = std::cmp::max(self.function_alignment, self.isa.symbol_alignment());
```

Comment:
I think you should default `function_alignment` to `isa.symbol_alignment()`.

Reply:
I didn't do that because it seemed that would allow a caller to override the function alignment to *smaller* than `isa.symbol_alignment()`, and that simply doesn't work on s390x.